[2025-06-29T18:01:01.712+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:00:00+00:00 [queued]>
[2025-06-29T18:01:01.721+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:00:00+00:00 [queued]>
[2025-06-29T18:01:01.721+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-29T18:01:01.732+0000] {taskinstance.py:2191} INFO - Executing <Task(DockerOperator): dbt_task> on 2025-06-29 18:00:00+00:00
[2025-06-29T18:01:01.736+0000] {standard_task_runner.py:60} INFO - Started process 469 to run task
[2025-06-29T18:01:01.739+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'films_dag_orchestaror', 'dbt_task', 'scheduled__2025-06-29T18:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/orchestrator.py', '--cfg-path', '/tmp/tmp5lnm9i2q']
[2025-06-29T18:01:01.741+0000] {standard_task_runner.py:88} INFO - Job 37: Subtask dbt_task
[2025-06-29T18:01:01.798+0000] {task_command.py:423} INFO - Running <TaskInstance: films_dag_orchestaror.dbt_task scheduled__2025-06-29T18:00:00+00:00 [running]> on host 9909ba106c7b
[2025-06-29T18:01:01.887+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='films_dag_orchestaror' AIRFLOW_CTX_TASK_ID='dbt_task' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-29T18:00:00+00:00'
[2025-06-29T18:01:01.919+0000] {docker.py:359} INFO - Starting docker container from image ghcr.io/dbt-labs/dbt-postgres:1.9.latest
[2025-06-29T18:01:01.927+0000] {docker.py:367} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2025-06-29T18:01:03.773+0000] {docker.py:429} INFO - [0m18:01:03  Running with dbt=1.9.0
[2025-06-29T18:01:04.067+0000] {docker.py:429} INFO - [0m18:01:04  Registered adapter: postgres=1.9.0
[2025-06-29T18:01:04.264+0000] {docker.py:429} INFO - [0m18:01:04  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
There are 1 unused configuration paths:
- models.my_dbt_project.example
[2025-06-29T18:01:04.422+0000] {docker.py:429} INFO - [0m18:01:04  Found 3 models, 3 sources, 431 macros
[2025-06-29T18:01:04.426+0000] {docker.py:429} INFO - [0m18:01:04
[2025-06-29T18:01:04.427+0000] {docker.py:429} INFO - [0m18:01:04  Concurrency: 4 threads (target='dev')
[2025-06-29T18:01:04.428+0000] {docker.py:429} INFO - [0m18:01:04
[2025-06-29T18:01:04.539+0000] {docker.py:429} INFO - [0m18:01:04  1 of 3 START sql table model dev.film_rating ................................... [RUN]
[2025-06-29T18:01:04.539+0000] {docker.py:429} INFO - [0m18:01:04  2 of 3 START sql table model dev.staging ....................................... [RUN]
[2025-06-29T18:01:04.705+0000] {docker.py:429} INFO - [0m18:01:04  2 of 3 OK created sql table model dev.staging .................................. [[32mSELECT 360[0m in 0.16s]
[2025-06-29T18:01:04.707+0000] {docker.py:429} INFO - [0m18:01:04  1 of 3 OK created sql table model dev.film_rating .............................. [[32mSELECT 360[0m in 0.17s]
[2025-06-29T18:01:04.710+0000] {docker.py:429} INFO - [0m18:01:04  3 of 3 START sql table model dev.film_actors ................................... [RUN]
[2025-06-29T18:01:04.914+0000] {docker.py:429} INFO - [0m18:01:04  3 of 3 OK created sql table model dev.film_actors .............................. [[32mSELECT 116640[0m in 0.20s]
[2025-06-29T18:01:04.932+0000] {docker.py:429} INFO - [0m18:01:04
[2025-06-29T18:01:04.932+0000] {docker.py:429} INFO - [0m18:01:04  Finished running 3 table models in 0 hours 0 minutes and 0.50 seconds (0.50s).
[2025-06-29T18:01:04.980+0000] {docker.py:429} INFO - [0m18:01:04
[2025-06-29T18:01:04.982+0000] {docker.py:429} INFO - [0m18:01:04  [32mCompleted successfully[0m
[2025-06-29T18:01:04.982+0000] {docker.py:429} INFO - [0m18:01:04
[2025-06-29T18:01:04.984+0000] {docker.py:429} INFO - [0m18:01:04  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-06-29T18:01:05.986+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=films_dag_orchestaror, task_id=dbt_task, execution_date=20250629T180000, start_date=20250629T180101, end_date=20250629T180105
[2025-06-29T18:01:06.017+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-29T18:01:06.043+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
